### DISCUSSION TITLE
QField not synchronizing nor downloading projects anymore

### DISCUSSION THREAD

**Discussion by rhernandezcemssa** [opengisch/QField#3592] [Bugs]
Hello, 
I have several cloud projects I've been working on from my desktop, I keep them synchronized with a mobile device via Qfieldsync daily as I edit them constantly, they consist of sets of vector layers and two raster files I use as offline base maps. Qfieldsync was working just fine yesterday, but ever since this morning it won't sync my projects on the mobile device, "network error" and "[QF/unknown_error] QFieldCloud Unknown Error" keep popping up when trying to download them on QField.

I've tried deleting some files like the heavy raster ones in case they are somehow corrupting the project, even attempted to create a new cloud project with very few light layers but nothing has worked out so far. Could it be a server issue?

**Comment by Daniel-ALTRI** [comment]
Same here, i work yesterday all day, as i do everyday on qfield. I try to sync yesterday nigth and until now i cant do it. Just appear QFieldCloud Unknown Error, or on PC 'Server Error (500)'.  I found an atualization on server at 21h yesterday, it should broke all servers and the sync. How can we do for dont lose all the work done in projet?

**Comment by strrings** [comment]
Also having this issue - we've been unable to push changes to QField Cloud. All changes are appearing as 'Error' and will not apply.

**Comment by suricactus** [comment]
We are well aware, see https://github.com/opengisch/QField/discussions/3594 . For updates check that link.

**Reply by suricactus** [reply]
@rhernandezcemssa @Daniel-ALTRI @strrings the service is now stabilized, see the ticket above for explanation.

**Reply by Daniel-ALTRI** [reply]
Now i can sign in my acount, but the sync still not working. I try new points on project and sync, but always Error:
![image](https://user-images.githubusercontent.com/117732505/200597872-a979ce79-9b8b-4133-9d18-e14c5cf781a3.png)

**Reply by rhernandezcemssa** [reply]
> @rhernandezcemssa @Daniel-ALTRI @strrings the service is now stabilized, see the ticket above for explanation.

Just checked, still can't download my projects to QField on the mobile device, getting the same error messages as before.

**Comment by turzik-x** [comment]
I have same problem with creating qgfield package on cloud. I think the problem is timeout is now 60s, not 600seconds. Can you revert timeout to 600seconds. The package process is terminated.

**Reply by aliasatp** [reply]
Where do you set the timeout? I don't figure out the download or package issue

**Reply by suricactus** [reply]
it's server side, good catch @turzik-x . I got reset today for some reason.

**Reply by suricactus** [reply]
Ok, found what was resetting the value of the worker timeout. It is related to the DNS failure from earlier today. A PR is coming and with the next version the problem will be solved.

**Reply by aliasatp** [reply]
it works! thanks

**Comment by aliasatp** [comment]
Hi all, at the moment I can't download projects to my mobile. I can push my data. So the issue is about downloading

**Reply by turzik-x** [reply]
I think is not about downloading but about preparing package for qfield. The process is failed, then downloading is failed too.

**Reply by aliasatp** [reply]
it could but, the message I receive is about downloading failure but in "jobs" I get "package" failed

**Comment by rhernandezcemssa** [comment]
QFfield working correctly again for me, tested downloading and syncing projects on the mobile app, syncing on the desktop with QFieldsync as well as making edits and uploading them to the cloud, all functioning as usual. 

Thank you @suricactus for cheking the issue, I hope it's working fine again for everyone else as well.

**Reply by suricactus** [reply]
Great to hear it works.

**Reply by Daniel-ALTRI** [reply]
All working perfect right now! Thanks @suricactus for support.

**Reply by joshmh** [reply]
Still not working for me. Maybe the project is bigger than average, so it's timing out? Anything you need to look into this?

**Comment by JNRosa1** [comment]
I still can't sync my qfield app. Everything ok in qgis. What can i do?
